sys-fs/tmsu/tmsu-0.6.1.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit golang-build
+
+EGO_PN="github.com/oniony/TMSU/"
+DESCRIPTION="Files tagger and virtual tag-based filesystem"
+HOMEPAGE="https://github.com/oniony/TMSU/wiki"
+SRC_URI="https://github.com/oniony/TMSU/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="test zsh-completion"
+
+RDEPEND=""
+DEPEND="
+	dev-go/go-sqlite3
+	dev-lang/go
+	dev-libs/go-fuse
+	zsh-completion? ( app-shells/zsh )
+"
+
+src_unpack() {
+	if [ "${A}" != "" ]; then
```

Comment:
Instead of manually running `unpack` yourself, you can call `default` so that the default unpack phase takes place, then you can perform `mv`